editor.planx.uk/src/@planx/components/shared/utils.ts

```diff
@@ -60,24 +59,31 @@ export const makeData = <T>(
  * // otherwise
  * // => "https://api.editor.planx.uk/test"
  */
-export const useStagingUrlIfTestApplication = (passport: Store.passport) => (
-  urlThatMightBeReplaced: string
-) => {
-  if (
-    [
-      passport.data?.[bopsDictionary.applicant_first_name],
-      passport.data?.[bopsDictionary.applicant_last_name],
-    ]
-      .map((x) => String(x).toLowerCase().trim())
-      .join("|") === "test|test"
-  ) {
-    const url = new URL(urlThatMightBeReplaced);
-    url.hostname = url.hostname.replace("planx.uk", "planx.dev");
-    return url.href;
-  }
+export const useStagingUrlIfTestApplication =
```

Comment:
Yep good thinking here - but was able to remove most all of these `||` cases with this update: https://github.com/theopensystemslab/planx-new/pull/1207#discussion_r1023151808

Definitely good thinking for future though, I don't expect this will be the last time we think about this issue!